app/components/common/FinalFooter.vue

```diff
@@ -15,31 +15,52 @@ export default Vue.extend({
       socialLinks: [
         {
           href: 'https://www.facebook.com/codecombat',
-          img: '/images/footer/facebook.svg',
+          img: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
```

Comment:
I see the idea of passing colors by defining them like this. But it basically duplicates svg code and makes it difficult to re-use. Can we instead create a vue component that takes color as prop?